components/CounterForm.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { useCart } from "../context/ItemsInCart";
+
+const CounterForm = ({ slug, name, isTiny }) => {
+  const { itemsInCart, setItemsInCart, addItem } = useCart();
+  const [amount, setAmount] = useState(0);
+
+  const decAmount = (int) => {
+    setAmount((amount) => amount - int);
+  };
+
+  const incAmount = (int) => {
+    setAmount((amount) => amount + int);
+  };
+
+  const handleSubmit = (event) => {
+    addItem({ name, slug, amount: amount });
+    event.preventDefault();
+    setAmount(() => 0);
+  };
+
+  const amountHandlerInputChange = (event) => {
+    setAmount(parseInt(event.target.value));
+    console.log(typeof amount);
+  };
+
+  /* 
+    - Input + 
+    Submit
+    
+    */
+
+  return (
+    <StyledForm onSubmit={handleSubmit} isTiny>
+      <fieldset>
+        <input type="button" onClick={() => decAmount(1)} value="-" />
+        <input
+          type="number"
+          min="0"
+          max="99"
+          step="1"
+          id="amount"
```

Comment:
setting the id to amount makes it so you'll have the same id multiple times whereas html defines id's to be unique